src/_sass/base.scss

```diff
@@ -253,3 +253,7 @@ input, textarea {
 hr {
   margin: $spacing-unit 0;
 }
+
+.circle-bullet {
+  list-style-type: circle;
```

Comment:
it seems weird to use different bullets than elsewhere on the site, e.g.
https://docs.tilt.dev/install.html